src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -242,4 +268,85 @@ private function getPrettyMetadata($type, $entity)
             return $meta ? '('.implode(', ', $meta).')' : '';
         }
     }
+
+    private function findWrongBundleOverrides(): array
+    {
+        $alternatives = array();
+        $bundleNames = array();
+
+        if ($this->rootDir) {
```

Comment:
I think we need to account for `$this->projectDir` being `null` here too.

Reply:
Done for this one and the next if too. Thank you